src/cook-web/src/app/globals.css

```
@@ -4,13 +4,28 @@
   font-size: 16px;
   -webkit-font-smoothing: subpixel-antialiased;
   -moz-osx-font-smoothing: auto;
 }
 
 :root {
+  /*
+   * Host UI typography. MarkdownFlow content and code blocks intentionally
+   * keep their own reading and monospace font contracts.
+   */
+  --font-app-sans:
+    system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)